src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/helpers/Messages.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.hmppstiertodeliusupdate.helpers
+
+fun tierUpdateMessage() = """
```

Comment:
There was a fair amount of work done on the definition of the message shape by Mark B so that it matches the others posted to the topic. It doesn't quite seem to have made it back into the ticket but here it is:

https://mojdt.slack.com/archives/CEUQKFNCS/p1615548204019500